bindings/python/tests/tester.py

```diff
@@ -400,11 +400,9 @@ def run(self):
                         inst.push(f)
                 elif inst.op == six.u("GET_ESTIMATED_RANGE_SIZE"):
                     begin, end = inst.pop(2)
-                    estimatedSize = obj.get_estimated_range_size_bytes(begin, end).wait()
```

Comment:
It's perhaps a bit unfortunate that we aren't using these values, but we do need to call these functions here. The intent of this code is that it is exercising the APIs, and I guess here we're just making sure that it doesn't produce an error. The same is true for other lines changed below this one in this file.

Reply:
I see, I'll change this and the next unused ones to an underscore to indicate that it is not being used.